service/src/LevelZeroDevicePool.cpp

```diff
@@ -164,6 +165,34 @@ namespace geopm
                                          dev_subdev_idx_pair.second);
     }
 
+    double LevelZeroDevicePoolImp::frequency_step(int domain, unsigned int domain_idx,
+                                                  int l0_domain) const
+    {
+        double result = NAN;
```

Comment:
Maybe add some kind of reminder that this is in MHz. You did already document that in the caller interface, but it could be helpful for future readers who come by this definition first. e.g., rename the function or rename `result` to something like `frequency_step_mhz` or add a comment reminding that this is not in Hz.